concepts/Common/Integration Guides/40 Bind DevExtreme components to GraphQL/40 Bind DevExtreme components to GraphQL.md

```diff
@@ -0,0 +1,335 @@
+## What is GraphQL?
+
+GraphQL is a trending API query language and data processing layer. Facebook engineers created GraphQL because REST APIs did not meet their performance requirements.
+
+Modern websites need to display large amounts of data from more than one source. To get this data from a REST API, applications have to issue multiple network requests. Each request takes time to resolve, which slows the application down. If you do not optimize your SQL queries, API responses may include unnecessary data. The application wastes time loading data users do not get to see.
+
+GraphQL is an intermediary layer that takes a single API request, and responds to it with data from multiple sources. It allows you to get very specific, and does not include unnecessary data in its responses. You can replace your REST API with GraphQL, or use GraphQL alongside your REST API.
+
+Follow this tutorial to try GraphQL for yourself.
+
+## Project overview
+
+This project consists of a client and a server.
+
+The client is a Vite/React Node.js application that displays a DevExtreme DataGrid. The server is an Express.js app that uses a GraphQL API to serve JSON data.
+
+The source code for this project is available [on GitHub](https://github.com/DevExpress-Examples/devextreme-datagrid-graphql-integration).
+
+## Step 1: First steps and global dependencies
+
+1. Initialize a Node.js project inside a new folder:
+
+        mkdir graphql-tutorial
+        cd graphql-tutorial
+        npm init
+
+
+2. Install project dependencies:
+
+        npm i express cors graphql-http graphql nodemon
+
+3. Copy the `data` folder from the [project repository](https://github.com/DevExpress-Examples/devextreme-datagrid-graphql-integration/tree/master/data) to your local project. The folder contains three `.json` files with static data. Our GraphQL server will use these files as its data sources.
+
+    ![Data sources](/images/GraphQL/data-sources.png)
+
+## Step 2: Client set-up
+
+The project's client side is a simple Vite application that displays a DevExtreme DataGrid.
+
+1. Create a `client` subfolder:
+
+        mkdir client
+        cd client
+
+2. Run the following command to create a new Vite project inside the folder:
+
+        npm create vite@latest
+
+3. Follow the wizard's instructions to create a **React** / **TypeScript** application named `vite-react`:
+
+    ![Vite wizard](/images/GraphQL/vite-create.png)
+
+4. Open the folder, and install the necessary dependencies for the Vite app:
+
+        cd vite-react
+        npm i @apollo/client devextreme devextreme-react graphql loadsh whatwg-fetch
+
+5. Copy the code from the [DataGrid CRUD demo](https://js.devexpress.com/React/Demos/WidgetsGallery/Demo/DataGrid/CRUDOperations/MaterialBlueLight/) to your Vite application.
```

Comment:
```suggestion
5. Copy the code from the [DataGrid CRUD demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/CRUDOperations/) to your Vite application.
```

I know that this article is React-only, but please do not use framework-dependent links to demos. Also always remove the theme.